.changeset/lovely-impalas-sell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-search-backend-module-explore': patch
+---
+
+Allows for an optional token manager to authenticate requests from the collator to the explore backend
```

Comment:
Can we add an example of how people might use this? The changesets get rolled up as release notes so having this information is really helpful for those upgrading